php/commands/site.php

```diff
@@ -749,34 +739,35 @@ public function delete_env($args, $assoc_args) {
     */
    public function deploy($args, $assoc_args) {
      $site = SiteFactory::instance( Input::site( $assoc_args ) );
-     $env = Input::env($assoc_args);
+     $env  = $site->environment(Input::env(
+       $assoc_args,
+       'env',
+       'Choose environment you want to initialize',
+       array('test', 'live')
+     ));
      $from = Input::env($assoc_args, 'from', "Choose environment you want to deploy from");
-     if (!isset($assoc_args['note'])) {
+     if(!isset($assoc_args['note'])) {
        $note = Terminus::prompt("Custom note for the Deploy Log", array(), "Deploy from Terminus 2.0");
```

Comment:
Can we at least align the variable names as "annotation", or at least "message"?